pkg/commands/drift/command.go

```diff
@@ -38,6 +36,12 @@ type DetectIamDriftCommand struct {
 	flagGCSBucketQuery        string
 	flagDriftignoreFile       string
 	flagMaxConcurrentRequests int64
+	flagSkipGitHubIssue       bool
+	flagGitHubToken           string
+	flagGitHubOwner           string
+	flagGitHubRepo            string
```

Comment:
In my implementation of Plan that i haven't submitted yet, I am pulling these GH specific variables from the context using https://github.com/sethvargo/go-githubactions.

Ive been fighting myself on the best way to do this accept as flag or use the environment.

Reply:
I think I prefer your way to using the context.

Reply:
Context/env var is weird to me. Especially since we are a cli tool there is no guarantee we will be run from a github repo/context with the right env var

Reply:
Hm, that might be true for the IAM drift, but plan/apply will be tightly coupled to running inside a pull request (creating comments, grouping outputs, etc).

Reply:
I think my preference would be to use a pattern like:

```
default := env.get('REPO')
f := flag.value('repo', default)
```

That way we can still use flags but we fallback to the environment. 

My NIT about https://github.com/sethvargo/go-githubactions is that this seems like a fairly heavyweight solution to just expose a couple of env variables. Additionally, I skimmed the README and it wasn't obvious to me that this was the right solution (improved documentation might help this).